_drafts/oryx/browser-compatibility.md

```diff
@@ -0,0 +1,24 @@
+# Supported browsers / viewports
+Oryx supports and is being tested on all modern browsers and operating systems:
+
+* iOS 16.x (Safari latest, Google Chrome latest)
+* Android 13.x (Google Chrome latest)
+* macOS Ventura (Safari latest, Google Chrome latest) 
+* Windows 10 and higher (Google Chrome latest, Edge latest)
+
+This allows us to cover more than 95% of operating systems and 93% of browsers used worldwide.
+
+---
+
+As for screen resolutions, we test our components on real devices: 
+
+* 393 x 786 px - iOS | iPhone 14 Pro Max | Safari
+* 360 x 670 px - Android | Samsung Galaxy S20 | Chrome
+* 1440+ px wide - macOS latest | Safari latest
```

Comment:
Not sure how useful is this number for customer? Is it 1440px? Or 1920? Or 100500? :)
Isn't it better to provide a range of supported resolutions/width? And how it's actually working?

Reply:
1440 and 1366 are standard screen widths for laptops :) 

they make more sense rather than abstract 992 or 1024px))

Reply:
done